### PR TITLE
feat(Logs in dashboards): Release note, minor doc updates

### PR DESCRIPTION
**Pull request by barbnewrelic**
For DOC-7960

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/ui-data/use-logs-ui.mdx [hunk 1]
```diff
@@ -195,16 +195,23 @@ Here are a few examples of how you can use the Logs UI to get detailed informati
...
-    You can add log charts to a dashboard directly from the Logs UI.
+    From the **Logs** UI, you can add log data as a [table widget](/docs/query-your-data/explore-query-data/use-charts/chart-types/#widget-table) to a dashboard. You can also create log tables with the data explorer or the query builder in New Relic.
 
     1. Go to **[one.newrelic.com](https://one.newrelic.com) > Logs**.
     2. Select a saved view. Or, search for results you want to plot; for example, `service_name:"checkout service" "process failed"`.
-    3. Click **Add to dashboard**, then fill out the details to add to an existing or new dashboard.
+    3. Click **Add to dashboard**, then fill out the details to add the log chart as a table to an existing or new dashboard.
 
+    
+    Then, from your dashboard you can:
```
Comment (paperclypse) [inline src/content/docs/logs/ui-data/use-logs-ui.mdx:207]:
```suggestion
Then, from your dashboard you can:
```

File: src/content/docs/release-notes/logs-release-notes/logs-22-05-18.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+subject: Logs
+releaseDate: '2022-05-18'
+version: '220518'
+---
+
+
+import logsDashboards from 'images/logs-dashboards-052722.png'
```
Comment (paperclypse) [inline src/content/docs/release-notes/logs-release-notes/logs-22-05-18.mdx:8]: I couldn't tell if any of the guids in this image needed to be blurred out?

**Review by paperclypse — DISMISSED** [review]
Looks good! I left a couple suggestions/questions, but I think it's probably fine to merge.

File: src/content/docs/release-notes/logs-release-notes/logs-22-05-18.mdx (hunk 2 above)
Comment (barbnewrelic) [reply src/content/docs/release-notes/logs-release-notes/logs-22-05-18.mdx:8]: No, it's Demotron. :-)

**Pull request merged**